async function greet() {
	throw "404 Error resolved";
	return "Hello, World!";
}

greet()
.then((result) => {
	console.log("promise was resolved");
	console.log(result);
})

.catch((error) => {
	console.log("promise was rejected");
	console.log(error);
});
